src/UI/Settings/NetImport/Options/NetImportOptionsView.js

```diff
@@ -20,7 +16,11 @@ var view = Marionette.ItemView.extend({
 	},
 	
 	onShow : function() {
+		var params = new URLSearchParams(window.location.search.slice(1));
```

Comment:
![image](https://cloud.githubusercontent.com/assets/23205277/22866487/6facf63c-f145-11e6-9e8c-fd4f219cff73.png)

ya sorry -- I always intended to swing back to this... just didnt have the chance yet -- will fix it now.